projects/uvis_gs_fail/data_process.py

```diff
@@ -0,0 +1,238 @@
+# Imports:
+import os
+import sys
+from multiprocessing import Pool
+import numpy as np
+import pandas as pd
+from astropy.io import fits
+import tqdm
+from skimage.transform import resize, rotate
+
+
+
+def log_image_process(fpath, orig_data=False):
+    '''
+    Opens a flt.fits file and logarithmically processes the data according to 
+    the following method:
+    - opens the flt file, extracts SCI array, and creates an array of image 
+    data.
+    - Sets all values in the image array that are less than one to be equal to 
+    1.
+    - Scales the image data logarithmically.
+    - Uses min/max scaling to scale the pixel values to be between 0 and 1. 
+    - Resizes the image array to the dimensions (256, 256).
+
+    Parameters:
+    -----------
+    fpath: string
+    The string that identifies the directory that the image is located in.
+
+    Returns: numpy array of floats
+    --------
+    log_data: array of floats
+    Array of logarithmically scaled image data.
+
+    rootname: string
+    A String containing the rootname of the image that was processed.
+    '''
+
+     #Unpacking the image data: 
```

Comment:
Make sure comments are on the same spacing as succeeding code in the script.
From
```
     #Unpacking the image data:
        # get the boolean value of the subarray
    subarray_value = fits.getval(fpath, 'subarray')
```
To
```
    # Unpacking the image data:
    # get the boolean value of the subarray
    subarray_value = fits.getval(fpath, 'subarray')
```